src/views/Static/index.tsx

```diff
@@ -0,0 +1,118 @@
+// External
+import React, { useState, useEffect } from 'react'
+import { useWallet } from 'use-wallet'
+import { useTheme } from 'styled-components'
+import { useTranslation } from 'react-i18next'
+import { useParams } from 'react-router-dom'
+import { useDispatch } from 'react-redux'
+import WalletConnector from 'cryptowalletconnector'
+// import styled from 'styled-components'
+
+// Hooks
+// import { useElementWidth } from 'src/hooks/useElementWidth'
+import { useWindowSize } from 'src/hooks/useWindowSize'
+import { useMountEffect } from 'src/hooks/useMountEffect'
+
+// Actions
+import { setPageTitle } from 'src/redux/page'
+
+// Components
+import { Header } from 'src/components/Header'
+import { Footer } from 'src/components/Footer'
+import { BackButton } from 'src/components/BackButton'
+import { StaticHeader } from '../Static/components/StaticHeader'
+import { StaticContent } from '../Static/components/StaticContent'
+import { Container } from 'src/components/Container'
+import { CardBody } from 'src/components/CardBody'
+import { MobileFooter } from 'src/components/MobileFooter'
+import { Card } from 'src/components/Card'
+import { Flex } from 'src/components/Flex'
+
+import { HeaderItem } from '../Auction/components/HeaderItem'
+
+// Svg
+import MetamaskImage from 'src/assets/svg/metamask.svg'
+import WalletImage from 'src/assets/svg/wallet_connect.svg'
+
+// Mesa Utils
+
+// Wallet Utils
+import { getRandomWallet } from 'src/utils/wallets'
+
+// Views
+// import { NotFoundView } from 'src/views/NotFound'
+
+
+interface StaticViewParams {
+  staticId: string
+}
+
+export function StaticView() {
+  const wallet = useWallet()
+  const { isMobile } = useWindowSize()
+
+  const walletAddress = wallet.account ? `${wallet.account.substr(0, 6)}...${wallet.account.substr(-4)}` : ''
```

Comment:
I have this repeated in many places. @light-fury can we refactor this into a utility function, please?